### PR TITLE
Update builds.md

### PR DESCRIPTION
**Pull request by icarusdust**

### CODE REVIEW

**Comment by icarusdust** [comment]
Fixes #446

File: content/rest-api/builds.md [hunk 1]
```diff
@@ -126,6 +126,16 @@ curl -H "Content-Type: application/json" -H "x-auth-token: <API Token>" --reques
...
 Returns the build information of an already running build on Codemagic.
 
+| **Status**  |
+| ------------- | 
+| `building`    | 
+| `finished`    | 
+| `failed`      | 
+| `queued`      | 
+| `skipped`     | 
+| `cancelled`   |
```
Comment (VeArnold) [inline content/rest-api/builds.md:136]: use American English in docs

cancelled > canceled

File: content/rest-api/builds.md [hunk 2]
```diff
@@ -126,6 +126,16 @@ curl -H "Content-Type: application/json" -H "x-auth-token: <API Token>" --reques
 
 Returns the build information of an already running build on Codemagic.
 
+| **Status**  |
```
Comment (VeArnold) [inline content/rest-api/builds.md:129]: Comprehensive list:
```
    QUEUED = "queued"
    PREPARING = "preparing"
    FETCHING = "fetching"
    TESTING = "testing"
    BUILDING = "building"
    PUBLISHING = "publishing"
    FINISHING = "finishing"

    FINISHED = "finished"
    WARNING = "warning"
    FAILED = "failed"
    CANCELED = "canceled"
    TIMEOUT = "timeout"
    SKIPPED = "skipped"
```

**Review by VeArnold — CHANGES_REQUESTED** [review]

File: content/rest-api/builds.md (hunk 2 above)
Comment (icarusdust) [reply content/rest-api/builds.md:129]: changes made.

Comment (mikhail-tokarev) [reply content/rest-api/builds.md:129]: maybe let's sort it by alphabet?

Comment (icarusdust) [reply content/rest-api/builds.md:129]: Yes, would be better. Ordered now.

**Review by VeArnold — APPROVED** [review]
LGTM

**Pull request merged**